snippets/base/migrations/0019_auto_20170725_1357.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0018_auto_20170614_0925'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='snippet',
+            name='campaign',
```

Comment:
Any idea  why this is here? You don't seem to have altered the `campaign` field.